.gitignore

```diff
@@ -395,3 +395,6 @@ build/override/
 # WindowsAppSDK specific files
 Microsoft.WinUI.AppX.targets
 !dev/vsix/**/*.pubxml
+
+# TAEF related
+redist/
```

Comment:
Why is this needed? It doesn't look like you changed anything about TAEF install, and this doesn't seem to have been an issue for others in the past.

Reply:
Thank you for your observations. Removed this change.